test/rendering/__fixtures__/scenario_10.fix.js

```diff
@@ -125,6 +125,9 @@ export default () => ({
         showDetails: false,
         showDetailsExpression: false,
         components: [
+          {
+            key: 'general',
```

Comment:
Since you are using `.find`, probably you dont need this!

Reply:
yeah this should have been here earlier and we would have caught this bug. But I added it anyway, might help us catch bugs in the future